### DISCUSSION TITLE
How can I use Zellij in iSH on iOS?

### DISCUSSION THREAD

**Discussion by ylorn** [zellij-org/zellij#2027] [Q&A]
Does zellij have plans to support Alpine Linux's [APK package manager](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management)?  was trying to install it to [iSH](https://apps.apple.com/us/app/ish-shell/id1436902243) but cannot find it with command `apk search`.

I assume that I can't directly use `zellij-x86_64-unknown-linux-musl`, because the name suggests it is compiled for `x86_64` instead of `arm_64`, which is iPhone/iPad's arch?

**Comment by imsnif** [comment]
Hey, the Zellij maintainers don't package Zellij for distributions. This is something done by third parties. You can take a look at your third party install document for more info: https://github.com/zellij-org/zellij/blob/main/docs/THIRD_PARTY_INSTALL.md

Also, you can always download a precompiled executable from our releases if you like.
